loaders/src/OBJ/babylon.objFileLoader.ts

```diff
@@ -264,14 +264,11 @@ module BABYLON {
             return this.importMesh(null, scene, data, rootUrl, null, null, null);
         }
 
-        public loadAssets(scene: Scene, data: string, rootUrl: string, onError?: (message: string, exception?: any) => void):Nullable<AssetContainer>{
+        public loadAssetContainer(scene: Scene, data: string, rootUrl: string, onError?: (message: string, exception?: any) => void): AssetContainer {
```

Comment:
Why this change?

Reply:
The name change is because it was inconsistent with what the SceneLoader called it. The return value change is because there is no reason why it should return null.

Reply:
ok!

Reply:
👍